react-basic-practice/src/components/CartBar/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Button from '../common/Button';
+import CartItem from '../CartItem';
+import closeButton from '@/assets/images/iconButton/btn-close.png';
+
+import './index.css';
+
+const data = [
+  {
+    idProduct: 2,
+    quantity: 3,
+    name: 'hal earrings',
+    price: 25,
+  },
+  {
+    idProduct: 1,
+    quantity: 4,
+    name: 'lira earrings',
+    price: 20,
+  },
+];
+
+const CartBar = () => {
+  // const {
+  //   onClose, data,
+  // } = props;
+  const onHandleDeleteCartItem = () => {};
```

Comment:
Don't put `on` and `handle` in the same name. Use `onDeleteCartItem` and `handleDeleteCartItem`.

Reply:
`onXXX` is used to define an event. `handleXXX` is used to define an event handler.